committing.rst

```diff
@@ -202,35 +202,19 @@ So a file name may be
 The contents of a news file should be valid reStructuredText. An 80 character
 column width should be used. There is no indentation or leading marker in the
 file (e.g. ``-``). There is also no need to start the entry with the issue
-number as it's part of the file name itself. Example news entry::
-
-   Fix warning message when ``os.chdir()`` fails inside
-   ``test.support.temp_cwd()``.  Patch by Chris Jerdonek.
-
-In addition to inline reST, news entries also support the usage of
-`Sphinx roles <https://devguide.python.org/documenting/#id4>`_. Several
-commonly used roles include :func:, :class:, and :meth:. When a
-corresponding entry is found within the documentation, the text within
-the role is converted into a hyperlink. Example news entry with Sphinx::
+number as it's part of the file name itself. You can use
+:ref:`inline markups <rest-inline-markup>` too. Example news entry::
 
    Fix warning message when :func:`os.chdir` fails inside
    :func:`test.support.temp_cwd`. Patch by Chris Jerdonek.
 
-The inline Sphinx roles can be used to assist readers in finding more
-information and context on the changes made. If the inline link does not
-provide additional context, an inline reST code block can be used instead::
-
-   ``<object>``
+The inline Sphinx roles like ``:func:`` can be used to assist readers in finding
```

Comment:
I do't think so.  Roles are documented well in https://devguide.python.org/documenting/#id4.  No need to additional links.

Reply:
> I do't think so. Roles are documented well in https://devguide.python.org/documenting/#id4.

That's why I was suggesting adding a link to https://devguide.python.org/documenting/#id4, similar to how you added a link for more information on reST:
``:ref:`inline markups <rest-inline-markup>` too. Example news entry::``

If you're entirely opposed to adding this one in though, it would probably be fine without the link.

Reply:
Also, it might not be entirely clear to readers of the devguide where to find https://devguide.python.org/documenting/#id4, since the title of the section is just "Inline markup" and it's fairly far down on the "Documenting Python" page. If they're looking for more information on what Sphinx roles we support, it might not be overly obvious.